docs/modules/reference/pages/billing/subscription/user-subscription.adoc

```diff
@@ -0,0 +1,119 @@
+
+The email used for registering does *not* need to match the email invitation was sent to.
+User can also choose to use their existing Google or GitHub account instead ❷.
+Signup will fail if user already has an account.
+In such case they need to choose option "Already have account? btn:[Log in]" ❶.
+
+Afterwards their invitation is accepted.
+
+image::billing/subscription/user-subscription/accepted.png[Screenshot of acceptance screen]
+
+btn:[Go To Subscription] will take them to link:../overview.html[Subscription Overview].
+
+User will be listed as _Active_ on the <<Manage Subscription Users Screen,user list screen>> after accepting.
+Accepting of invitation may fail if the user is already member of the subscription, even if they are <<Enabling or Disabling a User, disabled>>.
+
+=== Other Means of Inviting
+
+If invited user misses the invitation email, the menu action btn:[Resend Invite] allows for sending the email again and also to change the email address of user being invited.
+
+Clicking btn:[Copy Invitation Link] will bring you to page that lists the invitation page the user should follow.
+With that you can transfer link by other means, ie instant message.
+Invitation link is single use only, no further verification is performed when visiting it, therefore make sure no unauthorized actors get hold of it.
+Specifically, Payara Cloud Support will never ask you for invitation link.
+
+image::billing/subscription/user-subscription/copy-link.png[Screenshot of Copy Invitation Link]
+
+=== Editing a User
+
+Clicking btn:[Edit User] shows the properties of respective user of this subscription.
+
+image::billing/subscription/user-subscription/edit.png[Screenshot of Edit User screen]
+
+It is possible to change the role of the user on this screen.
+Updates need to be confirmed by clicking btn:[Update User].
+
+When updating the <<Role Requirements, requirements>> for role staffing need to be maintained, otherwise the update will fail.
+Only a Billing Manager can grant or remove Billing Manager roles in the subscription.
+
+New roles will be effective after next the user logs in the next time.
+
+==== Enabling or Disabling a User
+
+By un-clicking checkbox btn:[Active] the user will become _Inactive_.
+An inactive user does not have any permissions in the subscription.
+For them the effect is the same as if they were <<Removing a User, removed>>, it is however possible to restore their permissions by restoring btn:[Active] back to enabled state without need for inviting the user again.
+
+=== Removing a User
+
+To completely remove a user select btn:[Remove User] option from the action menu for the respective user.
+
+NOTE: The action cannot be undone.
+User will need to be invited again to gain access to subscription.
+To temporarily remove a user, consider <<Enabling or Disabling a User,disabling>> the user instead.
+
+The deletion need be confirmed:
+
```

Comment:
For usability, should the button be a big fat red one instead? At least I see in some apps such torpedo operations have big fat red buttons. Blue appears too peaceful for such a terminal operation IMHO

Reply:
Big fat red buttons are not part of our design language. Instead we have confirmation dialogs "You're about to do something potentially stupid".